### ISSUE TITLE
Add support for Kafka source

### ISSUE THREAD

**Issue by BenFradet** [snowplow/snowplow-elasticsearch-loader#18]
from snowplow/snowplow#2655

**Comment by BenFradet** [comment]
we may want to use https://github.com/confluentinc/kafka-connect-elasticsearch

**Comment by alexanderdean** [comment]
Yes, I'd like us to do a deep evaluation of that project as part of this ticket:

* Should we embedding code from Kafka Connect Elasticsearch (KCE) into the Snowplow Elasticsearch Loader (SEL)?
* Or should we be extracting SEL's core as a library, and then embedding that library into KCE, or into a fork of KCE?
* Or some other approach?

And perhaps some general questions about Kafka Connect:

* Are there patterns from Kafka Connect in general that we should be adopting across the Snowplow Loaders?
* Should we be looking at forking and extending Kafka Connect so it works with non-Kafka stream technologies?
